tutorials/08-voting/contracts/GovernanceToken.cdc

```diff
@@ -0,0 +1,200 @@
+import FungibleToken from "./FungibleToken.cdc"
+
+// GovernanceToken.cdc
+//
+// The GovernanceToken contract is a sample implementation of a fungible token on Flow which can be used for voting.
+
+pub contract GovernanceToken: FungibleToken {
+    // checkpointCounter is incremented with each proposal. 
+    // For voting, only the user's balance before 
+    // the respective proposal checkpoint value will be taken into account.
+    // TODO: this should not be public
+    pub(set) var checkpointCounter: UInt16
```

Comment:
Isn't this supposed to be a timestamp? Do you still want to make this an emulator/CLI tutorial instead of a playground tutorial?

Reply:
Yes! 

I actually did it in the very first draft: https://github.com/onflow/voting-tutorial/blob/main/tutorials/08-voting/contracts/GovernanceToken.cdc#L36

I changed after I was informed about the idea of merging this previous PR code into mine: https://github.com/onflow/flow-playground-tutorials/pull/8/files.

But in reality I would like to use timestamp. I will change to it.